<launch>



    <!-- Launch xbot2-core with the dummy -->
    <node pkg="repair_motion_controller" type="run_xbot2_dummy.py" name="xbot2_dummy" output="screen" />


    <!-- Launch the xbot_os_bridge node -->
    <node name="xbot_ros_bridge_node" pkg="repair_motion_controller" type="xbot_ros_bridge_node" output="screen" >
        
    </node>

    <!-- launch  the repair_motion_control_server.py -->
    <node name="repair_motion_controller" pkg="repair_motion_controller" type="repair_motion_control_server.py" output="screen" >
        <param name="enable_vis" type="bool" value="true" /> 
        <param name="robot" type="string" value="dummy" />   <!--  value options: { dummy, real } -->
    </node>


</launch>

